src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -759,8 +759,6 @@ private static ImmutableArray<RefKind> GetRefKindsOrNull(ArrayBuilder<RefKind> r
                 int p = (!argsToParamsOpt.IsDefault) ? argsToParamsOpt[a] : a;
                 var parameter = parameters[p];
 
-                Debug.Assert(!processedParameters.Contains(p));
```

Comment:
>Debug.Assert(!processedParameters.Contains(p)); [](start = 16, length = 47)

I think this assert should be put back. It catches real problems.

Reply:
The method should not be called in scenarios outlined in the issue.

---
In reply to: [177268255](https://github.com/dotnet/roslyn/pull/25730#discussion_r177268255) [](ancestors = 177268255)